README.md

```diff
@@ -1,11 +1,10 @@
-Terraform Provider
+Terraform Provider for Google Cloud Platform
 ==================
 
 - Website: https://www.terraform.io
 - [![Gitter chat](https://badges.gitter.im/hashicorp-terraform/Lobby.png)](https://gitter.im/hashicorp-terraform/Lobby)
 - Mailing list: [Google Groups](http://groups.google.com/group/terraform-tool)
 
-![Terraform](https://rawgithub.com/hashicorp/terraform/master/website/source/assets/images/logo-hashicorp.svg)
```

Comment:
The SVG is unbounded (so it will fill 100% of the container's width). You could use a real image tag:

```
<img src="https://cdn.rawgit.com/hashicorp/terraform-website/master/source/assets/images/logo-text.svg" width="600px">
```